src/PlusCommon/PlusCommon.h

```diff
@@ -47,13 +47,13 @@ enum PlusImagingMode
 
 /* Define case insensitive string compare for Windows. */
 #if defined( _WIN32 ) && !defined(__CYGWIN__)
-#  if defined(__BORLANDC__)
-#    define STRCASECMP stricmp
-#  else
-#    define STRCASECMP _stricmp
-#  endif
+  #if defined(__BORLANDC__)
+    #define STRCASECMP stricmp

```

Comment:
Right, gotcha. I view the STRCASECMP for usage with C style strings and new methods for C++ strings. I suppose compilers will automatically construct C++ from C. Still replace?